valhalla/baldr/double_bucket_queue.h

```diff
@@ -81,7 +81,6 @@ class DoubleBucketQueue {
    * Destructor.
    */
   virtual ~DoubleBucketQueue() {
-    clear();
```

Comment:
why do this in the destructor? it happens automatically

Reply:
Can you just remove the destructor altogether, now that it does nothing ?

Reply:
i think so

Reply:
ill make the class `final` as well just in case